cylc/flow/cfgspec/globalcfg.py

```diff
@@ -1978,24 +1978,29 @@ def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
 
     @classmethod
-    def get_inst(cls, cached: bool = True) -> 'GlobalConfig':
+    def get_inst(
+        cls, cached: bool = True, reload: bool = False
+    ) -> 'GlobalConfig':
         """Return a GlobalConfig instance.
 
         Args:
-            cached (bool):
-                If cached create if necessary and return the singleton
-                instance, else return a new instance.
+            cached:
+                If True, return a cached instance if present. If False, return
+                a new instance.
+            reload:
+                If true, reload the cached instance (implies cached=True).
+
```

Comment:
Worth guarding against future misuse?

```suggestion
                If true, reload the cached instance (implies cached=True).
    """
    if reload and cached is False:
        raise CylcError('You cannot reload an un-cached global config')
```

Reply:
I think it's ok for `reload=True` to override the `cache` argument

Reply:
Never mind - I don't think that this that important.